router/usecase/candidate_routes_test.go

```diff
@@ -35,26 +35,122 @@ func (s *RouterTestSuite) TestCandidateRouteSearcher_HappyPath() {
 		name          string
 		tokenIn       sdk.Coin
 		tokenOutDenom string
+		uniquePoolIDs map[uint64]struct{}
+		routes        int
 	}{
 		{
 			name:          "UOSMO -> USDT",
 			tokenIn:       sdk.NewCoin(UOSMO, one),
 			tokenOutDenom: USDT,
+			routes:        20,
+			uniquePoolIDs: map[uint64]struct{}{
+				1066: {},
+				1077: {},
+				1078: {},
+				1079: {},
+				1081: {},
+				1094: {},
+				1110: {},
+				1133: {},
+				1134: {},
+				1135: {},
+				1159: {},
+				1189: {},
+				1205: {},
+				1220: {},
+				1221: {},
+				1261: {},
+				1263: {},
+				1265: {},
+				1279: {},
+				1281: {},
+				1399: {},
+				1400: {},
+				1464: {},
+				1477: {},
+				1895: {},
+			},
```

Comment:
I think we should avoid hardcoding specific pool IDs. Not sure what exactly this assertion accomplishes

Otherwise, if we regen the state, the tests will break, requiring us to spend time modifying.